### PR TITLE
pkg/ocicni: handle if error channel is nil

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: pkg/ocicni/ocicni.go [hunk 1]
```diff
@@ -142,6 +142,9 @@ func (plugin *cniNetworkPlugin) monitorNetDir() {
 				logrus.Errorf("CNI setting failed, continue monitoring: %v", err)
 
 			case err := <-watcher.Errors:
```
Comment (rajatchopra) [inline pkg/ocicni/ocicni.go]: Does this really happen?

Comment (rajatchopra) [reply pkg/ocicni/ocicni.go]: I mean an event with nil value?

**Comment by vbatts** [comment]
It did on the Windows machine I'm testing on

-------- Original Message --------

File: pkg/ocicni/ocicni.go (hunk 1 above)
Comment (dcbw) [reply pkg/ocicni/ocicni.go]: @rajatchopra yes, it seems to, have reproduced on real clusters running RHEL too.

**Pull request merged**